Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:1.13
+
+WORKDIR /go/src/github.com/ministryofjustice/analytical-platform-auth0-user-exports
+
+COPY . .
+
+RUN go get -d -v ./...
+RUN go install -v ./...
+
+CMD ["analytical-platform-auth0-user-exports"]
```

Comment:
You may want to do something similar to [what we did in the unidler](https://github.com/ministryofjustice/analytics-platform-go-unidler/blob/master/Dockerfile). Basically use a stage for the build and one with just the binary and nothing else to keep the image to the minimum.